### PR TITLE
Kat category selection

### PR DESCRIPTION
**Pull request by maliciamrg**
select the category "anime" for anime and the category "tv" for serie for the kat provider.
and in the  contributing.md  the command i use to follow the guide was "git checkout -b develop origin/develop" instead of "git checkout develop"

### CODE REVIEW

**Comment by SiCKRAGEB0T** [comment]
Thanks for the pull request! Before a real human comes by, please make sure your report has all the below criteria checked

- [x] Give a description on what the PR is for.
- [x] Make sure you send a PR against the DEVELOP branch
- [x] Don't send big changes in one go. Split up big PRs into multiple smaller (easier manageble) PRs

Please make sure you also read [contribution guide](https://github.com/SiCKRAGETV/SickRage/blob/master/contributing.md) and followed all the steps.

Thanks!

File: contributing.md [hunk 1]
```diff
@@ -107,7 +107,7 @@ Please follow this process; it's the best way to get your work included in the p
 
    ```bash
    # Switch to the develop branch
-   git checkout develop
+   git checkout -b develop origin/develop
```
Comment (OmgImAlexis) [inline contributing.md]: @miigotu isn't this incorrect?

Comment (miigotu) [inline contributing.md]: Yes, this shouldnt be changed. Both ways will work the same IF develop branch doesnt already exist. Modern git will auto set up a new branch for you.

And the file was changed from 644 to 755 == no-no

File: sickbeard/providers/kat.py [hunk 2]
```diff
@@ -72,6 +72,14 @@ def _doSearch(self, search_strings, search_mode='eponly', epcount=0, age=0, epOb
...
         items = {'Season': [], 'Episode': [], 'RSS': []}
 
+#       select the correct category
+        if self.show.anime:
+            self.search_params.update({'category': 'anime'})
+            logger.log("search param 'category': 'anime' instead of 'tv' ", logger.DEBUG)
+        else:
+            self.search_params.update({'category': 'tv'})
+            logger.log("search param 'category': 'tv' ", logger.DEBUG)
+
```
Comment (miigotu) [inline sickbeard/providers/kat.py]: Just do:
`self.search_params.update({'category': ('tv', 'anime')[self.show.anime]})`

But you also need to make the change to the rss update url when self.show.anime

**Comment by miigotu** [comment]
Ive fixed this up and merging from #3062

**Comment by maliciamrg** [comment]
Thanks for the fix.

`<anime = (self.show and self.show.anime) or (epObj and epObj.show and epObj.show.anime) or False>` just blow my mind ,i'm so far of the `_doSearch` parameter complexity

**Pull request closed without merging**